owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/network/ChunkFromFileRequestBody.java

```diff
@@ -113,19 +108,10 @@ public void writeTo(BufferedSink sink) {
                 }
             }
 
-            Log.d(TAG, "Chunk with size " + mChunkSize + " written in request body");
+            Log_OC.v("Chunk with size $mChunkSize written in request body");
```

Comment:
This is not a kotlin file yet 😢, so 
`Log_OC.v("Chunk with size $mChunkSize written in request body");`
is not the same as
`Log_OC.v("Chunk with size " + mChunkSize + " written in request body");`

Reply:
solved